iis-wf/iis-wf-referenceextraction/src/test/resources/eu/dnetlib/iis/wf/referenceextraction/softwareurl/data/webcrawler/nosource/report.json

```diff
@@ -1,3 +1,3 @@
 {"key":"processing.referenceExtraction.softwareUrl.webcrawl.fromCache.total", "type":"COUNTER", "value": "0"}
-{"key":"processing.referenceExtraction.softwareUrl.webcrawl.processed.total", "type":"COUNTER", "value": "2"}
+{"key":"processing.referenceExtraction.softwareUrl.webcrawl.processed.total", "type":"COUNTER", "value": "0"}
```

Comment:
Was this change deliberate or was it introduced just because the module started to return `0` instead of `2`? ;)

This is one of the 3 comments about the same issue: `processed.total`, at least in my opinion, should sum both successfully processed records and `processed.fault`.